lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -415,6 +418,10 @@ def connectionsCustomControlCreator(aeTemplate, c):
         return None
 
 def arrayCustomControlCreator(aeTemplate, c):
+    # Note: UsdGeom.Tokens.xformOpOrder is a exception we want it to display normally.
+    if c == UsdGeom.Tokens.xformOpOrder:
+        return None
```

Comment:
Keep xformOpOrder as exception in that we don't want it to display as an array custom control. But just as a generic string attribute (so you can see the string value).